### PR TITLE
return only subproject segments in member profile

### PR DESCRIPTION
**Pull request by yeganathan18**
# Changes proposed ✍️

### What
copilot:summary
​
copilot:poem

### How
copilot:walkthrough

## Checklist ✅
- [x] Label appropriately with `Feature`, `Improvement`, or `Bug`.
- [ ] Add screenshots to the PR description for relevant FE changes
- [ ] New backend functionality has been unit-tested.
- [ ] API documentation has been updated (if necessary) (see [docs on API documentation](https://docs.crowd.dev/docs/updating-api-documentation)).
- [x] [Quality standards](https://github.com/CrowdDotDev/crowd-github-test-public/blob/main/CONTRIBUTING.md#quality-standards) are met.

### CODE REVIEW

File: backend/src/database/repositories/memberRepository.ts [hunk 1]
```diff
@@ -1558,6 +1559,7 @@ class MemberRepository {
           lfxMemberships: true,
           identities: false,
           segments: true,
+          OnlySubProjects: true,
```
Comment (sausage-todd) [inline backend/src/database/repositories/memberRepository.ts:1562]: Not a good way of naming a variable. Should start with a lowercase letter instead.

Comment (sausage-todd) [inline backend/src/database/repositories/memberRepository.ts:1562]:
```suggestion
          onlySubProjects: true,
```

File: backend/src/database/repositories/memberRepository.ts [hunk 2]
```diff
@@ -2233,11 +2235,13 @@ class MemberRepository {
       include = {
         identities: true,
         segments: false,
+        OnlySubProjects: false,
```
Comment (sausage-todd) [inline backend/src/database/repositories/memberRepository.ts:2238]:
```suggestion
        onlySubProjects: false,
```

File: backend/src/database/repositories/memberRepository.ts [hunk 3]
```diff
@@ -2233,11 +2235,13 @@ class MemberRepository {
...
         lfxMemberships: false,
         memberOrganizations: false,
       } as {
         identities?: boolean
         segments?: boolean
+        OnlySubProjects?: boolean
```
Comment (sausage-todd) [inline backend/src/database/repositories/memberRepository.ts:2244]:
```suggestion
        onlySubProjects?: boolean
```

**Review by sausage-todd — APPROVED** [review]
Looks good other than the variable name

File: backend/src/database/repositories/memberRepository.ts (hunk 1 above)
Comment (yeganathan18) [reply backend/src/database/repositories/memberRepository.ts]: Oops, I missed that.

Comment (yeganathan18) [reply backend/src/database/repositories/memberRepository.ts]: Fixed and updated the PR, thanks for the review! ✌🏻

**Pull request merged**